<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgba(170, 170, 255,.6);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:pad, x1:0.403, y1:0.3125, x2:1, y2:1, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(169, 255, 185, 255));</string>
  </property>
  <widget class="QLabel" name="fname_srch">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>700</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

font: 75 bold 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Fill only the field you want to update.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="est_cost">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>81</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);

font:  14pt &quot;Segoe UI &quot;;</string>
   </property>
   <property name="text">
    <string>Estimated Cost:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="estd_cost_edit">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>81</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit:focus{
background-color: rgb(255, 255, 255);
}
QTextEdit{
background-color: rgb(255, 255, 255);
	font: 12pt &quot;Segoe UI&quot;;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="discount">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>132</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);

font:  14pt &quot;Segoe UI &quot;;</string>
   </property>
   <property name="text">
    <string>Discount:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="discount_edit">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>132</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit:focus{
background-color: rgb(255, 255, 255);
}
QTextEdit{
background-color: rgb(255, 255, 255);
	font: 12pt &quot;Segoe UI&quot;;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updt_file_btn">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>305</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
background-color: rgb(170, 255, 255);
}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Update File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>355</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{


font:  14pt &quot;Segoe UI &quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;}
QPushButton:hover{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}
QPushButton:focus{
	background-color: rgb(161, 161, 161);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="desc_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="est_cost_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);

font:  14pt &quot;Segoe UI &quot;;</string>
   </property>
   <property name="text">
    <string>Desciption:</string>
   </property>
  </widget>
  <zorder>fname_srch</zorder>
  <zorder>est_cost</zorder>
  <zorder>estd_cost_edit</zorder>
  <zorder>discount</zorder>
  <zorder>discount_edit</zorder>
  <zorder>label_3</zorder>
  <zorder>updt_file_btn</zorder>
  <zorder>label_6</zorder>
  <zorder>cancel_btn</zorder>
  <zorder>desc_edit</zorder>
  <zorder>est_cost_2</zorder>
 </widget>
 <tabstops>
  <tabstop>estd_cost_edit</tabstop>
  <tabstop>discount_edit</tabstop>
  <tabstop>desc_edit</tabstop>
  <tabstop>updt_file_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
